skyportal/tests/api/test_sources.py

```diff
@@ -125,6 +126,7 @@ def test_starlist(manage_sources_token, public_source):
     assert isinstance(data['data']["starlist_info"][2]["dec"], float)
 
 
+@pytest.mark.xfail
```

Comment:
```suggestion
@pytest.mark.xfail(strict=False)
```
We want to insure that this test will not fail the suite if this function actually succeeds (which it normally does).